opentracing-spring-rabbitmq-it/src/test/java/io/opentracing/contrib/spring/rabbitmq/RabbitMqTracingAutoConfigurationWithoutRabbitTemplateItTest.java

```diff
@@ -126,6 +126,7 @@ RabbitTemplateProvider rabbitTemplateProvider(
       final CachingConnectionFactory cachingConnectionFactory =
           new CachingConnectionFactory(rabbitConnectionFactoryBean.getObject());
       TracingRabbitTemplate rabbitTemplate = new TracingRabbitTemplate(cachingConnectionFactory, tracer, spanDecorator);
+      rabbitTemplate.setUseDirectReplyToContainer(false);
```

Comment:
Required because newer spring-rabbit has introduced the direct reply to container feature and defaults to it. But we need the capability to override this functionality for the test as in TracingRabbitTemplate.doSendAndReceiveWithTemporary. 

The RabbitTemplate.doSendAndReceiveWithDirect is not overridable.